src/test/lib/import_export/csv/csv_parser_test.cpp

```diff
@@ -0,0 +1,275 @@
+#include "base_test.hpp"
```

Comment:
1:1 copy, only replaced operator execution with static method call and removed the test for the correct job execution (see CsvParser for details).